transforms/helpers/transform-helper.js

```diff
@@ -189,9 +230,17 @@ function createClassProp(j, instanceProp) {
  */
 function createActionDecoratedProps(j, actionsProp) {
   const actionProps = get(actionsProp, "value.properties");
-  const actionDecorators = createActionDecorators(j);
+  const overriddenActions = get(actionsProp, "overriddenActions") || [];
+  const actionDecorators = createIdentifierDecorators(j);
   return actionProps.map(actionProp =>
-    createMethodProp(j, actionProp, actionDecorators)
+    createMethodProp(
+      j,
+      actionProp,
+      actionDecorators,
+      actionsProp.runtimeType &&
```

Comment:
Not quite sure what `runtimeType` is being used for here or in the `createMethodProp` function. In what cases do we not have a `runtimeType`?

Reply:
If the `codemods` are run without the `runtime-data` we don't have runtime type. 

Also for some reason data could not be extracted for a file, though it is very unlikely

Reply:
I think this should probably be a bit more explicit then, currently it's hard to understand what's going on here. Maybe `hasRuntimeData` or something along those lines.

Reply:
Done, added a getter `hasRuntimeData`